tests/integration/cassettes/TestLiveThread_test_init.json

```diff
@@ -0,0 +1,164 @@
+{
+  "http_interactions": [
+    {
+      "recorded_at": "2016-12-03T04:01:48",
+      "request": {
+        "body": {
+          "encoding": "utf-8",
+          "string": "grant_type=client_credentials"
+        },
+        "headers": {
+          "Accept": "*/*",
+          "Accept-Encoding": "identity",
+          "Authorization": "Basic <BASIC_AUTH>",
```

Comment:
Did you have any issues or have to find a reference to be able to set the environment variables to get these secrets removed?